### ISSUE TITLE
indent rules not working in v0.5.0

### ISSUE THREAD

**Issue by acalvo** [linthtml/linthtml#245]
With LintHTML v0.4.2 and this `.linthtmlrc.js`:

```js
module.exports = {
  "indent-style": "spaces",
  "indent-width": 2
}
```

If I had this HTML file:

```html
<p>
<span>a</span>
</p>
```

`npx linthtml index.html` would report the following error:

```
✔ Checking rules config
✔ Found 1 files
✔ Analysing files

index.html
 2:1  error  Incorrect indentation for `span` beginning at L2:C1.  indent-style 
              Expected `<span>` to be at an indentation of 2 but                
             was found at 0.                                                    

✖ 1 problem (1 error, 0 warning)
```

That's OK. But if now I upgrade it to v0.5.0, and run `npx linthtml index.html` again, the error is gone 😕:

```
✔ Found 1 files
✔ Files analyzed

✨  There's no problem, good job 👏
```

**Comment by KamiKillertO** [comment]
🤔 Ok it seems that there's an error during the parsing of HTML that prevents the file from been linted correctly.
